source/fundamentals/encrypt-fields.txt

```diff
@@ -1,3 +1,12 @@
 .. _java-fle:
 
 .. sharedinclude:: dbx/encrypt-fields.rst
+
+   .. replacement:: driver-specific-content
+
+      .. important:: Compatible Encryption Library Version
+
+         The {+driver-short+} uses the `mongodb-crypt
```

Comment:
This is good, but let's also be explicit about the full coordinates of the dependency, and how to add it to both Maven and Gradle (like we do [here](https://www.mongodb.com/docs/drivers/java/sync/current/quick-start/#add-mongodb-as-a-dependency)).

Reply:
Sure, I can add the tabs into this note!